phpstan.neon.dist

```diff
@@ -2,7 +2,8 @@ parameters:
     level: max
 
     paths:
+        - src/
         - test/types/
```

Comment:
Didn't widen this to include the entire test directory because that would add `1752` errors to solve. This PR alone was already almost 1K.